docs-src/stargate-core/modules/install/partials/docker_compose.adoc

```diff
@@ -1,20 +1,41 @@
 // tag::3x[]
-=== New Cassandra and Stargate using docker-compose
+=== New Cassandra and Stargate using docker-compose 
+
+Using `docker-compose`, you can also start an Apache Cassandra^(TM)^ cluster or use one that is running in Docker already, to create a Cassandra plus Stargate environment. 
+This method is different from the standalone environment, in that a separate Cassandra cluster is used for the backend storage, and is more like a production environment.
 
 The `docker-compose` YAML file contains the configuration for pulling the
 required docker images and configuration required for installing:
 
 * 1-3 Apache Cassandra^(TM)^ {cass-tag-3x} nodes as a cluster (backend-1, backend-2, backend-3)
-* 1 Stargate node (stargate)
+* v2: 1 Stargate coordinator, plus desired APIs (by default, all are installed)
+* v1: 1 Stargate node (stargate) The Stargate image contains the Cassandra Query Language (CQL), REST, Document, GraphQL APIs, and GraphQL Playground.
 
-The Stargate image contains the Cassandra Query Language (CQL), REST, Document, GraphQL APIs,
-and GraphQL Playground.
+There is a shell script to run the docker-compose file with
+delays between node startup that prevents timing issues.
 
-Two companion files are helpful: a script to run the docker-compose file with
-delays between node startup that prevents timing issues, and a text file that
-has the `docker run` commands for running a CQLSH Docker container to query against
-either one of the Cassandra nodes or the Stargate node, respectively.
+// LLP 09.28.2022 NEED TO ADD BACK IN A SEPARATE SECTION FOR THIS IN THE DOCKER-COMPOSE directory
+//  and a text file that has the `docker run` commands for running a CQLSH Docker container to query against either one of the Cassandra nodes or the Stargate node, respectively.
+
+.v2
+[%collapsible%open]
+===== 
+Use this docker-compose https://github.com/stargate/stargate/blob/v2.0.0/docker-compose/cassandra-3.11/start_cass_311.sh[shell script] to start the coordinator and APIs in developer mode.
+The easiest way to do that is to navigate to the `<install_location>/stargate/stargate/docker-compose` directory, and run the script.
+Currently, Stargate v2 is in beta.
+You will want to run, for example:
+
+[source,bash,subs="attributes+"]
+----
+./start_cass_{cass-alt-tag-3x}.sh -t {stargate2-docker-tag}
+----
+
+The latest available beta image can be found https://hub.docker.com/r/stargateio/coordinator-3_11/tags[here].
```

Comment:
can remove the word `beta` so we don't have to change this when we go to GA

Reply:
fixed